src/content/docs/mobile-monitoring/new-relic-mobile/mobile-sdk/mobile-sdk-api-guide.mdx

```diff
@@ -0,0 +1,363 @@
+---
+title: Mobile SDK guide
+tags:
+  - Mobile monitoring
+  - API guides
+metaDescription: 'Customize your mobile agent instrumentation to send custom data and metrics to New Relic.'
+---
+
+Use the New Relic mobile SDK API calls to customize and extend the data your mobile app reports to New Relic. 
+
+While our mobile monitoring solution offers powerful out-of-the-box data collection, you may need to customize and extend the data your mobile app reports to New Relic. For example, you can use the public SDK API calls to:
+
+    * Start and stop interaction traces from events in your mobile app
+    * Record custom metrics
+    * Send custom attributes and events
+    * Track network requests from libraries not supported automatically
+    * Set custom IDs to user sessions
+
+Most of these customizations are available for both native (iOS and Android) and hybrid (React Native, Xamarin, etc.) mobile apps.
+
+## Get started [#get-started]
+
+1. Make sure you have the latest mobile agent installed.
+2. Review our guidelines for using iOS and Android SDK methods:
```

Comment:
@ally-sassman What will I get out of taking the time to review these guidelines?